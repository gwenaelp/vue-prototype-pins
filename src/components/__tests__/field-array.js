import Vue from "vue";
import FieldArray from "../field-array";

describe("field-array.vue", () => {
  const Constructor = Vue.extend(FieldArray);
  const vm = new Constructor().$mount();

  test("should match the snapshot", () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
