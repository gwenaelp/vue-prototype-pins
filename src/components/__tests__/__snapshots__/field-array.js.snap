// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`field-array.vue should match the snapshot 1`] = `<!---->`;
